File: app/back/src/main/java/net/seijishikin/jp/normalize/manage/kanrensha/dto/user/SaveKanrenshaSeijidantaiCapsuleDto.java
package net.seijishikin.jp.normalize.manage.kanrensha.dto.user;

import java.io.Serializable;

import net.seijishikin.jp.normalize.common_tool.dto.FrameworkCapsuleDto;
import net.seijishikin.jp.normalize.manage.kanrensha.dto.kanrensha.KanrenshaSeijidantaiDto;

/**
 * 関連者政治団体格納Dto
 */
public class SaveKanrenshaSeijidantaiCapsuleDto extends FrameworkCapsuleDto implements Serializable {

    /** Serialize id */
    private static final long serialVersionUID = 1L;

    /** 関連者政治団体Dto */
    private KanrenshaSeijidantaiDto kanrenshaSeijidantaiDto = new KanrenshaSeijidantaiDto();

    /**
     * 関連者政治団体Dtoを取得する
     *
     * @return 関連者政治団体Dto
     */
    public KanrenshaSeijidantaiDto getKanrenshaSeijidantaiDto() {
        return kanrenshaSeijidantaiDto;
    }

    /**
     * 関連者政治団体Dtoを設定する
     *
     * @param kanrenshaSeijidantaiDto 関連者政治団体Dto
     */
    public void setKanrenshaSeijidantaiDto(final KanrenshaSeijidantaiDto kanrenshaSeijidantaiDto) {
        this.kanrenshaSeijidantaiDto = kanrenshaSeijidantaiDto;
    }

}

File: app/front/src/components/main/dto/kanrensha/saveKanrenshaSeijidantaiCapsuleDto.ts
import { FrameworkCapsuleDto, type FrameworkCapsuleDtoInterface } from "seijishikin-jp-normalize_common-tool";
import { KanrenshaSeijidantaiDto, type KanrenshaSeijidantaiDtoInterface } from "./kanrenshaSeijidantaiDto";

interface SaveKanrenshaSeijidantaiCapsuleDtoInterface extends FrameworkCapsuleDtoInterface {

    /** 関連者政治団体Dto */
    kanrenshaSeijidantaiDto: KanrenshaSeijidantaiDtoInterface;
}



class SaveKanrenshaSeijidantaiCapsuleDto extends FrameworkCapsuleDto
    implements SaveKanrenshaSeijidantaiCapsuleDtoInterface {

    /** 関連者政治団体Dto */
    kanrenshaSeijidantaiDto: KanrenshaSeijidantaiDtoInterface;

    constructor() {
        super();

        this.kanrenshaSeijidantaiDto = new KanrenshaSeijidantaiDto();
    }
}

export { type SaveKanrenshaSeijidantaiCapsuleDtoInterface, SaveKanrenshaSeijidantaiCapsuleDto }
